docs/schema.md

```diff
@@ -16,7 +16,7 @@
 Desired:
 ```json
 {
-  "longhaulTelemetrySendInterval": 10
+  "longhaulTelemetrySendIntervalInSeconds": 11
```

Comment:
Could probably drop the `longhaul` prefix as everything in this repo is long-haul, right?